ocaml/xapi/storage_migrate.ml

```diff
@@ -542,6 +542,10 @@ let start' ~task ~dbg ~sr ~vdi ~dp ~url ~dest =
     error " Caught exception %s:%s. Performing cleanup." Api_errors.sr_not_attached sr_uuid;
     perform_cleanup_actions !on_fail;
     raise (Api_errors.Server_error(Api_errors.sr_not_attached,[sr_uuid]))
+  | Storage_interface.Backend_error(code, _) when code = "SR_BACKEND_FAILURE_44" ->
+    error "Caught exception SR_BACKEND_FAILURE_44. Performing cleanup.";
```

Comment:
It would be good to add some identifiers to the log message: `sr:%s vdi:%s url:%s dest:%s`.

Reply:
It is a good suggestion, as to provide the complete data for further debugging.  I checked the function, in the beginning, there is already a line covering all the information such as sr/vdi. So I think it is enough. Duplicating those information will make the log file large and not necessary.

Previous debug information is like:
debug "Mirror.start sr:%s vdi:%s url:%s dest:%s" sr vdi url dest;